### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by Aniruddh25**
Clarify some points in `CONTRIBUTING.md`

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -16,25 +16,26 @@ We very much welcome issues to help us improve the project. When submitting an i
...
 
 ### Running locally
 
 Before running the code, ensure you have the correct version of .NET installed (refer to the [global.json](global.json) file) and open the solution from `src/Azure.DataApiBuilder.Service.sln` in Visual Studio (or other editor of choice).
 
-The next step is to ensure you have a config file for DAB defined. If you want to use the sample one [you can use the generator tool](#an-alternative-way-to-generate-config-files), otherwise, ensure that the `dab-config.<engine>.json` file is in the `src/Service` directory (and if the database is CosmosDb_NoSql the GraphQL schema file too).
+The next step is to ensure you have a config file for DAB defined. You can pick a sample one from the `src/Service.Tests` project and copy the `dab-config.<engine>.json` file to the `src/Service` directory (and if the database is CosmosDb_NoSql the GraphQL schema file too). You can also use the [generator tool](#an-alternative-way-to-generate-config-files) described below. Note these sample configuration files rely on the backend schema to be as per the respective `src/Service.Tests/DatabaseSchema-<engine>.sql` schema file in the `src/Service.Tests` project.
```
Comment (seantleonard) [inline CONTRIBUTING.md:25]:
```suggestion
The next step is to ensure you have a config file for DAB defined. You can pick a sample one from the `src/Service.Tests` project and copy the `dab-config.<engine>.json` file to the `src/Service` directory (and if the database is CosmosDb_NoSql the GraphQL schema file too). You can also use the [generator tool](#an-alternative-way-to-generate-config-files) described below. Note these sample configuration files expect the backend database schema to be created from the respective `src/Service.Tests/DatabaseSchema-<engine>.sql` schema file in the `src/Service.Tests` project.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -16,25 +16,26 @@ We very much welcome issues to help us improve the project. When submitting an i
...
 
 Make sure the config has a valid connection string, or you use the [`@env` syntax to reference an environment variable](https://learn.microsoft.com/azure/data-api-builder/configuration-file#accessing-environment-variables).
 
 #### Visual Studio
 
-Before running the code in Visual Studio you must be sure to have your config file correctly populated with a valid connection string to your configured back end. Once you have the back end running and your configuration file set correctly you may do the following to run the code in Visual Studio without using the CLI.
+Before running the code in Visual Studio you must be sure to have your config file is correctly populated with a valid connection string to your configured back end. Once you have the back end running and your configuration file set correctly you may do the following to run the code in Visual Studio without using the CLI.
```
Comment (seantleonard) [inline CONTRIBUTING.md:31]:
```suggestion
Before running the sample code in Visual Studio, you must be sure to have your config file correctly populated with a valid connection string for your configured back end. Once you have the back end running and your configuration file set correctly, you may do the following to run the code in Visual Studio without using the CLI.
```

**Review by seantleonard — COMMENTED** [review]
Thanks for adding clarity to the project's contributor onboarding process!

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -53,8 +54,8 @@ Primarily DAB uses integration tests to verify the engine operates correctly in
 To run the SQL tests locally you will need to:
 
 1. Setup a database using the server(s) that you want to test against.
-1. Create the database schema using the `DatabaseSchema-<engine>.sql` file.
-1. Set the connection string in `dab-config.<engine>.json` or use the [`@env` syntax to reference an environment variable](https://learn.microsoft.com/azure/data-api-builder/configuration-file#accessing-environment-variables).
+1. Create the database schema using the `src/Service.Tests/DatabaseSchema-<engine>.sql` file.
+1. Set the connection string in `src/Service.Tests/dab-config.<engine>.json` or use the [`@env` syntax to reference an environment variable](https://learn.microsoft.com/azure/data-api-builder/configuration-file#accessing-environment-variables).
```
Comment (AbhijithGanesh) [inline CONTRIBUTING.md:58]: Should this all be listed as `1.` ?

Comment (AbhijithGanesh) [reply CONTRIBUTING.md:58]: Just checked this ! It is rendering 1,2,3 correctly!

Comment (seantleonard) [reply CONTRIBUTING.md:58]: Neat markdown feature :)

**Review by seantleonard — APPROVED** [review]
lgtm!

**Review by aaronburtle — APPROVED** [review]
LGTM!

**Pull request merged**